roq-core/src/main/java/org/roqmessaging/zookeeper/RoQZKHelpers.java

```diff
@@ -18,29 +26,125 @@ public static void createZNode(CuratorFramework client, String path) {
 		try {
 			client.create().forPath(path);
 		} catch (Exception e) {
-			// e.printStackTrace();
+			e.printStackTrace();
 		}
 	}
 	/**
 	 * Allow to create multiple znodes 
-	 * in an atomic fashion
+	 * in a transaction
 	 * @param client
+	 * @param pathsHCMBuMonitorToAdd 
+	 * @param pathHCMMonitorToAdd 
 	 * @param path
 	 */
 	public static void createQueueZNodes(CuratorFramework client, String queuePath,
-			String monitorPath, String monitorPL, String statMonitorPath, 
-			String statMonitorPL, String hcmPath, String hcmPL) {
+			String monitorPath, String monitorPL, String statMonitorPath,
+			String statMonitorPL, String hcmPath, String hcmPL, String scalingPath, String backupMonitorsPath,
+			List<Metadata.BackupMonitor> backupMonitors, String pathHCMMonitorToAdd, ArrayList<String> pathsHCMBuMonitorToAdd) {
 		try {
-			client.inTransaction().create().forPath(queuePath)
+			CuratorTransactionBridge transaction = client.inTransaction().create().forPath(queuePath)
+				.and().create().forPath(pathHCMMonitorToAdd)
 				.and().create().forPath(monitorPath, monitorPL.getBytes())
 				.and().create().forPath(statMonitorPath, statMonitorPL.getBytes())
 				.and().create().forPath(hcmPath, hcmPL.getBytes())
+				.and().create().forPath(scalingPath)
+				.and().create().forPath(backupMonitorsPath);
+			for (int i = 0; i < backupMonitors.size(); i++) {
+				transaction.and().create().forPath(makePath(backupMonitorsPath, backupMonitors.get(i).zkNodeString()), 
+						(backupMonitors.get(i).getData()).getBytes());
+			}
+			for (String pathBUToAdd : pathsHCMBuMonitorToAdd) {
+				transaction.and().create().forPath(pathBUToAdd);
+			}
+			transaction.and().commit();
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+	}
+	
+	/**
+	 * replace a monitor by a standby monitor inside a single
+	 * transaction
+	 * @param client
+	 * @param hCMStateAdd
+	 * @param hCMStateRemove
+	 * @param pathQBUMonitorToRemove
+	 * @param hcmPath
+	 * @param hcmAddress
+	 * @param statMonitorPath
+	 * @param statMonitor
+	 * @param monitorPath
+	 * @param monitor
+	 */
+	public static void replaceMonitorZNodes(CuratorFramework client,
```

Comment:
Please correct this comment: add `scalingPath` at the end. And What does _qbu_ in `pathQBUMonitorToRemove` mean?